### ISSUE TITLE
Can't load keras model (Unsupported keras layer type BatchNorm)

### ISSUE THREAD

**Issue by zzz3000** [deeplearning4j/deeplearning4j#7515]
#### Issue Description
I'm trying to import keras model for Mask RCNN and get the error 

org.deeplearning4j.nn.modelimport.keras.exceptions.UnsupportedKerasConfigurationException: Unsupported keras layer type BatchNorm. Please file an issue at http://github.com/deeplearning4j/deeplearning4j/issues.
	at org.deeplearning4j.nn.modelimport.keras.utils.KerasLayerUtils.getKerasLayerFromConfig(KerasLayerUtils.java:318)
	at org.deeplearning4j.nn.modelimport.keras.KerasModel.prepareLayers(KerasModel.java:218)
	at org.deeplearning4j.nn.modelimport.keras.KerasModel.<init>(KerasModel.java:164)
	at org.deeplearning4j.nn.modelimport.keras.KerasModel.<init>(KerasModel.java:96)
	at org.deeplearning4j.nn.modelimport.keras.utils.KerasModelBuilder.buildModel(KerasModelBuilder.java:307)
	at org.deeplearning4j.nn.modelimport.keras.KerasModelImport.importKerasModelAndWeights(KerasModelImport.java:144)
	at ImportKeras.main(ImportKeras.java:19)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.codehaus.mojo.exec.ExecJavaMojo$1.run(ExecJavaMojo.java:282)
	at java.lang.Thread.run(Thread.java:748)

#### Version Information
* Deeplearning4j version 1.0.0-beta3
* Linux 64
* no CUDA

**Comment by maxpumperla** [comment]
`BatchNorm`, as opposed to `BatchNormalization`, a standard Keras layer is a custom layer used in your RCNN example. I vaguely remember we've had this issue reported before.

The point is: we don't automatically support 3rd party extensions of Keras. We do support most built-in layers and the possibility to register lambdas. Beyond that, it would require a custom Java implementation of this layer, plus a mapping.
